src/elements/common/annotator-context/types.ts

```diff
@@ -28,10 +28,12 @@ export interface AnnotatorState {
 export type GetMatchPath = (history: History) => match<MatchParams> | null;
 
 export interface AnnotatorContext {
+    annotationsFetchError: Error | null;
```

Comment:
Does this need to be a separate error? Is there a way to leverage the `error` already part of `state`?